plugins/vendor/webex/infra/kWebexAPIClient.php

```diff
@@ -121,21 +123,32 @@ protected function saveNextPageLinkFromHeaders($responseHeaders)
 		}
 	}
 	
-	public function getRecordingsList($startTime, $endTime)
+	protected function convertTimeToWebexFormat($time)
 	{
 		$dateFormat = 'Y-m-d';
 		$hourFormat = 'H:i:s';
-		$startDate = gmdate($dateFormat, $startTime);
-		$startHour = gmdate($hourFormat, $startTime);
-		$endDate = gmdate($dateFormat, $endTime);
-		$endHour = gmdate($hourFormat, $endTime);
-		$request = "admin/recordings?from=$startDate" . "T$startHour" . "&to=$endDate" . "T$endHour";
+		$formattedDate = gmdate($dateFormat, $time);
+		$formattedHour = gmdate($hourFormat, $time);
+		return $formattedDate . 'T' . $formattedHour;
+	}
+	
+	public function getRecordingsList($startTime, $endTime)
+	{
+		$formattedStartTime = $this->convertTimeToWebexFormat($startTime);
+		$formattedEndTime = $this->convertTimeToWebexFormat($endTime);
+		$request = "admin/recordings?from=$formattedStartTime&to=$formattedEndTime";
+		return $this->sendRequest($request);
+	}
+	
+	public function getMeetingRecordingsList($meetingId)
+	{
+		$request = "admin/recordings?meetingId=$meetingId";
 		return $this->sendRequest($request);
 	}
 	
-	public function sendRequestUsingDirectLink($directLink)
+	public function sendRequestUsingDirectLink($directLink, $decodeJson = true)
```

Comment:
do we decode the response even if it was not really json?
also its either way defaults to true and not seems to be change in this function so dont think its really needed in the decleration